app/lib/pre_assembly/batch.rb

```diff
@@ -20,6 +20,7 @@ class Batch
              :object_manifest_rows,
              :staging_location_with_path,
              :progress_log_file,
+             :using_file_manifest,
              :project_name,
```

Comment:
refactor: make this database model attribute available so it can be used elsewhere in the code